app/controllers/wms_controller.rb

```diff
@@ -1,4 +1,6 @@
 class WmsController < ApplicationController
+  skip_before_action :verify_authenticity_token
```

Comment:
I had to add this to get Rails 5.1 to stop complaining about CSRF issues locally (no auth token).  Specifically, this was for the dynamic maps where points/areas dynamically populate the table.

So, yeah a Rails thing.  Not leaflet.  I figured since the controller only exposes the R in CRUD, it wasn't a big deal skip validation and make Rails happy.